src/palm-react-native-ui-kit/components/ProfileHeader.tsx

```diff
@@ -5,85 +5,137 @@ import { ContractAddr, SupportedNetworkEnum } from 'palm-core/types'
+          member.userId === sdk?.currentUser?.userId
+        )
+      })
+      if (checkOperator) {
+        setIsOperator(true)
+      }
+    }, [])
+
     const { profile, lensProfile } = useProfile({ profileId: userProfileId! })
     const profileImg = getProfileMediaImg(profile?.picture)
 
+    const { top } = useSafeAreaInsets()
+
     return (
       <View style={styles.container}>
-        <View style={styles.header}>
+        <View
+          style={[
+            styles.header,
+            {
+              height: 168 + top,
+            },
+          ]}
+        >
           <ImageBackground
             source={{ uri: profile?.coverPicture }}
             resizeMode="cover"
             style={{ flex: 1 }}
           >
             {isMyPage ? (
-              <View style={{ alignItems: 'flex-end' }}>
+              <View style={{ marginTop: top, alignItems: 'flex-end' }}>
                 <Row style={styles.headerButtons}>
                   <Pressable
                     style={styles.headerButton}
                     onPress={(): void => {
                       navigation.navigate(Routes.UpdateProfile)
                     }}
                   >
-                    <Icon name={'pencil'} size={24} />
+                    <Ionicons name={'pencil'} size={28} />
                   </Pressable>
                   <Pressable
                     style={styles.headerButton}
                     onPress={(): void => {
                       navigation.navigate(Routes.Setting)
                     }}
                   >
-                    <Icon name={'settings-outline'} size={24} />
+                    <Ionicons name={'settings-outline'} size={28} />
                   </Pressable>
                 </Row>
               </View>
             ) : (
-              <View style={{ alignItems: 'flex-start' }}>
+              <View
+                style={{
+                  marginTop: top,
+                  flexDirection: 'row',
+                  justifyContent: 'space-between',
+                }}
+              >
                 <Pressable
                   style={styles.headerButton}
                   onPress={(): void => {
                     navigation.goBack()
                   }}
                 >
-                  <Icon
+                  <Ionicons
                     name="ios-chevron-back"
                     color={COLOR.black._800}
-                    size={24}
+                    size={28}
                   />
                 </Pressable>
+                {isOperator && (
+                  <Pressable
+                    style={styles.headerButton}
+                    onPress={(): void => {
+                      onToggleChannelUserControl?.()
+                    }}
+                  >
+                    <MaterialIcons
+                      name="more-horiz"
+                      color={COLOR.black._800}
+                      size={28}
+                    />
+                  </Pressable>
```

Comment:
why are you doing this in `ProfileHeader` This is a general profile header used in user profile pages

Reply:
When a user viewing the profile is an operator and accessed through the group channel, should have to can mute or ban users.

Reply:
I think we should be able to do it from the channel members list view. Let's not do it in the generic profile header. weird UX and code.